### PR TITLE
[no ticket] pass query_project instead

### PR DESCRIPTION
**Pull request by Qi77Qi**

### CODE REVIEW

**Review by yonghaoy — APPROVED** [review]
This work work here. but for places like 
https://github.com/verily-src/workbench-examples/blob/d7e02600eb0f29c36d51b5a086bf04464fe7bde5/cromwell_setup/gvs_wdls/untested_wdls/GvsCreateFilterSet.wdl#L72
we would need more work. (but I am not sure if AoU will be using it)

**Review by jmthibault79 — COMMENTED** [review]
[edit: this was the experience after the first commit only]

Running the workflow with this version does advance past this job, but it later fails at GvsExtractCallset -> GetNumSamplesLoaded

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 1]
```diff
@@ -112,15 +112,15 @@ workflow GvsExtractCallset {
 
   call Utils.GetBQTableLastModifiedDatetime as SamplesTableDatetimeCheck {
     input:
-      project_id = project_id,
+      project_id = query_project,
```
Comment (yonghaoy) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:115]: Line 18, 
`String query_project = project_id`
I think we need to update the upstream method and passing the correct query_project

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 2]
```diff
@@ -809,12 +809,11 @@ task GatherVariantCallingMetrics {
     disks: "local-disk ${disk_size_gb} HDD"
     bootDiskSizeGb: 15
     preemptible: 1
-    noAddress: true
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:812]: is this change necessary?

**Review by mcovarr — COMMENTED** [review]
kicked off an integration test for these changes

File: scripts/variantstore/wdl/GvsExtractCallset.wdl (hunk 2 above)
Comment (Qi77Qi) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:812]: I thought this was reverted..

@yonghaoy

Comment (yonghaoy) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:812]: ah, I throught i removed it. trying again

**Review by yonghaoy — APPROVED** [review]

**Review by mcovarr — APPROVED** [review]
